### PR TITLE
Adding FHIR B2C ARM template samples

### PR DESCRIPTION
**Pull request by msjasteppe**

### CODE REVIEW

**Comment by msjasteppe** [comment]
@shellyhaverkamp please take a look at this PR.

**Comment by shellyhaverkamp** [comment]
#assign-reviewer: @EXPEkesheth

File: samples/fhir-aad-b2c/oauth2Permissions.json [hunk 1]
```diff
@@ -0,0 +1,280 @@
+{
+	"oauth2Permissions": [
+		{
```
Comment (namalu) [inline samples/fhir-aad-b2c/oauth2Permissions.json:3]: Can you delete this first JSON element in the array? We are not supporting patient.all.all.

```json
{
	"adminConsentDescription": "Allows user to read, write and delete all resources in a patient's compartment.",
	"adminConsentDisplayName": "patient.all.all",
	"id": "0d1a5f28-49cb-4a16-97f5-106efd659dc8",
	"isEnabled": true,
	"lang": null,
	"origin": "Application",
	"type": "User",
	"userConsentDescription": "Allows user to read, write and delete all resources in a patient's compartment.",
	"userConsentDisplayName": "patient.all.all",
	"value": "patient.all.all"
}
```

Comment (msjasteppe) [reply samples/fhir-aad-b2c/oauth2Permissions.json:3]: Completed.

**Review by namalu — APPROVED** [review]

File: samples/fhir-aad-b2c/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# ARM templates for FHIR service integration with Azure Active Directory B2C
+
+You can use the FHIR® service in Azure Health Data Services with Azure Active Directory B2C (Azure AD B2C). This capability gives healthcare organizations a secure and convenient way to grant access to the FHIR service with fine-grained access control for different users or groups, without creating or commingling user accounts in their organization’s Microsoft Entra ID tenant.
```
Comment (shellyhaverkamp) [inline samples/fhir-aad-b2c/README.md:3]:
```suggestion
You can use the FHIR® service in Azure Health Data Services with Azure Active Directory B2C (Azure AD B2C). This capability gives healthcare organizations a secure and convenient way to grant access to the FHIR service with fine-grained access control for different users or groups, without creating or comingling user accounts in their organization’s Microsoft Entra ID tenant.
```

**Pull request merged**